jdi-light-parallel-examples/src/main/java/io/github/epam/entities/User.java

```diff
@@ -0,0 +1,9 @@
+package io.github.epam.entities;
+
+import com.epam.jdi.tools.DataClass;
+
+public class User extends DataClass<User> {
+    public String name = "Roman", password = "Jdi1234", lastName,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use one line for each declaration, it enhances code readability.](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5133577)